test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -89,6 +89,14 @@ class test_ucp_rma : public test_ucp_memheap {
     }
 
     void test_message_sizes(blocking_send_func_t func, size_t *msizes, int iters, int is_nbi);
+
+    void conditionally_disable_test() {
+        // TODO: need to investigate the slownees of the disabled tests
+        if ((GetParam().transports.front().compare("dc_x") == 0) &&
```

Comment:
can we add this to init() function? to avoid changing all test functions

Reply:
good catch! done